notebooks/MAST/PanSTARRS/PS1_DR2_TAP/PS1_DR2_TAP.ipynb

```diff
@@ -0,0 +1,899 @@
+    "    pylab.subplot(511+i)\n",
+    "    w = np.where((detection_TAP_results['filterType']==filter) & (detection_TAP_results['psfQfPerfect']>0.9))[0]\n",
+    "    pylab.plot(t[w],dmag[w],'-o')\n",
+    "    pylab.ylabel('{} [mag - {:.2f}]'.format(filter,magmean[w[0]]))\n",
+    "    pylab.xlim(xlim)\n",
+    "    pylab.ylim(ylim)\n",
+    "    if i==0:\n",
+    "        pylab.title(objname)\n",
+    "pylab.xlabel('Time [MJD]')\n",
+    "pylab.tight_layout()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot versus the periodic phase instead of epoch\n",
+    "\n",
+    "Plot versus phase using known RR Lyr period from Simbad (table [J/AJ/132/1202/table4](http://vizier.u-strasbg.fr/viz-bin/VizieR-3?-source=J/AJ/132/1202/table4&-c=KQ%20UMa&-c.u=arcmin&-c.r=2&-c.eq=J2000&-c.geom=r&-out.max=50&-out.form=HTML%20Table&-oc.form=sexa))."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "period = 0.48636\n",
+    "# convert flux in Jy to magnitudes\n",
+    "t = (detection_TAP_results['obsTime'] % period) / period\n",
+    "mag = -2.5*np.log10(detection_TAP_results['psfFlux']) + 8.90\n",
+    "magmean = 0.0*mag\n",
+    "for i, filter in enumerate([b'g',b'r',b'i',b'z',b'y']):\n",
+    "    magmean[detection_TAP_results['filterType']==filter] = TAP_results[filter.decode('ascii')+'MeanPSFMag'][0]\n",
+    "dmag = mag - magmean\n",
+    "dmag1 = dmag[detection_TAP_results['psfQfPerfect']>0.9]\n",
+    "# fix the x and y axis ranges\n",
+    "xlim = np.array([t.min(),t.max()])\n",
+    "xlim = xlim + np.array([-1,1])*0.02*(xlim[1]-xlim[0])\n",
+    "# flip axis direction for magnitude\n",
+    "ylim = np.array([dmag1.max(),dmag1.min()])\n",
+    "ylim = ylim + np.array([-1,1])*0.02*(ylim[1]-ylim[0])\n",
+    "\n",
+    "pylab.rcParams.update({'font.size': 14})\n",
+    "pylab.figure(1,(10,10))\n",
+    "for i, filter in enumerate([b'g',b'r',b'i',b'z',b'y']):\n",
+    "    pylab.subplot(511+i)\n",
+    "    w = np.where((detection_TAP_results['filterType']==filter) & (detection_TAP_results['psfQfPerfect']>0.9))[0]\n",
+    "    w = w[np.argsort(t[w])]\n",
+    "    pylab.plot(t[w],dmag[w],'-o')\n",
+    "    pylab.ylabel('{} [mag - {:.2f}]'.format(filter,magmean[w[0]]))\n",
+    "    pylab.xlim(xlim)\n",
+    "    pylab.ylim(ylim)\n",
+    "    if i==0:\n",
+    "        pylab.title(objname)\n",
+    "pylab.xlabel('Phase')\n",
+    "pylab.tight_layout()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Repeat search using eclipsing binary KIC 2161623\n",
+    "\n",
+    "From [Villanova Kepler Eclipsing Binaries](http://keplerebs.villanova.edu)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "objname = 'KIC 2161623'\n",
+    "ra, dec = resolve(objname)\n",
+    "radius = 1.0/3600.0 # radius = 1 arcsec\n",
```

Comment:
If you use the mast astroquery resolver this would change to:
```
coords = Mast._resolve_object(objname)
ra,dec = coords.ra.value, coords.dec.value
```